Dockerfile

```diff
@@ -1,11 +1,22 @@
 FROM php:7.1-apache-stretch
 
+ENV LD_LIBRARY_PATH /usr/local/instantclient
+
+# Setup the required extensions.
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install NVM and the current (as of 26/02/2019) LTS version of Node.
+ENV NVM_DIR /usr/local/nvm
+ENV NODE_VERSION  lts/carbon
```

Comment:
Trivial, an extra blank space before the value of the env var

Reply:
Fixed